content/yaml/building-a-flutter-app.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Building a Flutter app
+description: Building a Flutter app with YAML.
+weight: 2
+---
+
+With `codemagic.yaml`, you can use Codemagic to build, test and publish Flutter apps. You can read more about how to use `codemagic.yaml` and see the structure of the file [here](../yaml/yaml).
+
+## Android builds
+
+The following templates show code signing using `key.properties`.
+
+### Building .apk with default debug code signing
```

Comment:
All codesigning stuff move to code signing and publishing page

Reply:
fixed in 5bb0ce0378dce6344e9f190dbd98bf19958cace1